framework/tlvf/tlvf.py

```diff
@@ -1057,6 +1094,8 @@ def addClassVarLenMethods(self, obj_meta, param_type, param_name, param_meta, pa
                 lines_cpp.append( "%sm_%s_idx__++;" % (self.getIndentation(1), param_name) )
                 if is_var_len:
                     lines_cpp.append( "%sif (!m_parse__) { (*m_%s)++; }" % (self.getIndentation(1), param_length) )
+            else: 
+                lines_cpp.append("%sm_%s_init = true;" %(self.getIndentation(1),param_meta.name))
```

Comment:
please add a comment here which explains this - refer to the function which adds the member that you will add later.